doc/nrf/installation/install_ncs.rst

```diff
@@ -104,6 +104,9 @@ Depending on your preferred development environment, complete the following step
 
    .. group-tab:: nRF Connect for Visual Studio Code
 
+      .. note::
+            If you prefer, you can now `start VS Code walkthrough`_ and install the toolchain and the SDK from there.
+
```

Comment:
suggest adding this one (as well / instead of here) before "In Visual Studio Code, the latest version of the [nRF Connect for VS Code Extension Pack]".
I see much value this link providing during installation as it allows to install extension in just 2 clicks instead of 4, which adds multiple different screens to navigate and read through

Reply:
Applied. FYI, the whole section for prerequisites will be further reworked before NCS 3.0.0, so the link placement and phrasing there will change.